From 62caed13764d52c4ec938e5f69efe3979bdcf6ad Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 17 Jan 2024 16:48:05 +0100
Subject: [PATCH 477/841] refactor(stm32mp2): remove unused variable

bl32_mem_params is not used in function bl2_plat_handle_post_image_load,
remove it.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git fe019d9eccfa1fc430c3a5dcceea11990a166850 refactor(stm32mp2): remove unused variable]
Change-Id: I57d70bc716d66cec2e80f428c62f843a3f376893
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/352183
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 plat/st/stm32mp2/bl2_plat_setup.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/plat/st/stm32mp2/bl2_plat_setup.c b/plat/st/stm32mp2/bl2_plat_setup.c
index f871d13fc..7f238b8cf 100644
--- a/plat/st/stm32mp2/bl2_plat_setup.c
+++ b/plat/st/stm32mp2/bl2_plat_setup.c
@@ -506,7 +506,6 @@ int bl2_plat_handle_post_image_load(unsigned int image_id)
 {
 	int err = 0;
 	bl_mem_params_node_t *bl_mem_params = get_bl_mem_params_node(image_id);
-	bl_mem_params_node_t *bl32_mem_params __unused;
 	bl_mem_params_node_t *pager_mem_params __unused;
 	const struct dyn_cfg_dtb_info_t *config_info;
 	unsigned int i;
-- 
2.47.2

